.github/workflows/win.yml

```
@@ -38,210 +38,34 @@ jobs:
             libs/mpir.lib
             libs/yaml.lib
             libs/xml2.lib
-          key: win-libs-${{ hashFiles('.github/workflows/win.yml') }}-msvc-${{ env.VSCMD_VER }}
-      - name: Download libgc
-        if: steps.cache-libs.outputs.cache-hit != 'true'
-        uses: actions/checkout@v3
-        with:
-          repository: ivmai/bdwgc
-          ref: v8.2.2
-          path: bdwgc
-      - name: Download libatomic_ops
-        if: steps.cache-libs.outputs.cache-hit != 'true'
-        uses: actions/checkout@v3
-        with:
-          repository: ivmai/libatomic_ops
-          ref: v7.6.14
-          path: bdwgc/libatomic_ops
+          key: win-libs-${{ hashFiles('.github/workflows/win.yml') }}-${{ hashFiles('etc/win-ci/*.ps1') }}-msvc-${{ env.VSCMD_VER }}
```

Haven't tried, but this should work per [docs](https://docs.github.com/en/actions/learn-github-actions/expressions#hashfiles)

```suggestion
          key: win-libs-${{ hashFiles('.github/workflows/win.yml', 'etc/win-ci/*.ps1') }}-msvc-${{ env.VSCMD_VER }}
```